flask_cors/__init__.py

```diff
@@ -381,12 +418,14 @@ def _set_cors_headers(resp, options):
 
     # If CORS has already been evaluated via the decorator, skip
     if hasattr(resp, FLASK_CORS_EVALUATED):
+        logger.debug('CORS have been already evaluated, skipping')
         return resp
 
     headers_to_set = _get_cors_headers(options,
                                        request.headers,
                                        request.method,
                                        resp.headers)
+    logger.debug('CORS headers to be set {}'.format(str(headers_to_set)))

```

Comment:
I think it would be great to bump this up to an INFO level, and perhaps change the tensing to "Settings CORS headers: ...". Thoughts?